maggy/core/executors/tf_dist_executor.py

```diff
@@ -0,0 +1,361 @@
+#
+#   Copyright 2021 Logical Clocks AB
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+
+import builtins as __builtin__
+import inspect
+import json
+import traceback
+import os
+import random
+import socket
+from typing import Callable, Any, Tuple
+
+import numpy as np
+import tensorflow
+import tensorflow as tf
```

Comment:
choose one

Reply:
The _init_seed function it is actually never called. I'll delete it